File: Lv1/이상한문자만들기.java
public class 이상한문자만들기 {

    public String solution(String s) {

        StringBuilder sb = new StringBuilder();

        int i = 0 ;
        for(char ch : s.toCharArray()) {
            if(ch == ' ') {
                sb.append(' ');
                i = 0;
                continue;
            }

            if(i == 0 || i % 2 == 0) {
                sb.append(Character.toUpperCase(ch));
            }
            else {
                sb.append(Character.toLowerCase(ch));
            }
            i++;
        }

        return sb.toString();
    }

}

File: kotlin/Lv1/이상한문자만들기.kt
package kotlin.Lv1

class 이상한문자만들기 {

    fun solution(s: String): String {
        val sArray = s.split(" ")
        var answer = ""
        var index = 0

        for(str in sArray) {
            answer += createWeirdString(str)
            index++
            if(index != sArray.size) {
                answer += " "
            }
        }

        return answer
    }

    fun createWeirdString(str: String): String {
        val weirdString = StringBuilder()
        var index = 0
        for(ch in str) {
            if(index % 2 == 0) {
                weirdString.append(ch.uppercaseChar())
            } else {
                weirdString.append(ch.lowercaseChar())
            }
            index++
        }
        return weirdString.toString()
    }

}